.github/workflows/ci-front.yaml

```diff
@@ -130,6 +130,8 @@ jobs:
         run: cd front && yarn
       - name: Front / Run linter
         run: cd front && yarn lint
+      - name: Front / Run linter for CI
+        run: cd front && yarn lint --config .eslintrc-ci.js
```

Comment:
Hi @alfredlouisw8, thank you for contributing!

Instead of adding a step and linting twice with two different eslint configs, I think it would be simpler and faster to only use `.eslintrc-ci.js` in the CI.

So here is what I suggest:
- Merge the `Front / Run linter` and `Front / Run linter for CI` steps. Only keep `Front / Run linter` but use `.eslintrc-ci.js`.
- Make `.eslintrc-ci.js` extend `.eslintrc.js`. For this we need to add `extends: ["./.eslintrc.js"],` to `.eslintrc-ci.js`. `.eslintrc-ci.js` will then check all rules from `.eslintrc.js`, and then add the `no-console` rule.